lib/rules/declaration-property-value-keyword-no-deprecated/README.md

```diff
@@ -0,0 +1,84 @@
+# declaration-property-value-keyword-no-deprecated
+
+Disallow deprecated keywords for properties within declarations.
+
+<!-- prettier-ignore -->
+```css
+  a { color: ThreeDDarkShadow; }
+/**     ↑         ↑
+ * property and value pairs like these */
+```
+
```

Comment:
```suggestion

This rule flags keywords that were removed or deprecated after being in the CSS specifications, including Editor Drafts, and were either:

- shipped in a release browser version
- shipped in a pre-release browser version
- shipped but behind feature flags
- polyfilled with some adoption before any browser shipped
- given an MDN page
```

Let's add these criteria to the extended description.

Reply:
Iv removed the last point because I consider it not enough until we resolve #7510.

e.g.
If [`:blank`](https://developer.mozilla.org/en-US/docs/Web/CSS/:blank#browser_compatibility) ever gets removed, only a polyfill would warrant it to be detected by `selector-no-deprecated`—not just an MDN page.
see w3c/csswg-drafts#1967

Reply:
I suggested including `given an MDN page` because I've encountered this multiple times :)

I would find something on MDN, try it and find that it didn't work or wasn't on a standards track anymore.

Dev rel for browser engines will often contribute to MDN for features they are prototyping. It happens that the docs land, but the feature does not.

Or MDN authors will work from a spec that only has a partial implementation. For MDN authors it isn't always immediately clear what has or hasn't shipped and they might include too much too quickly.

Inclusion on MDN makes such features very visible to CSS authors and is a strong signal that something can be used. Even when the docs sometimes do not reflect the reality of implementations.

Authors might still leave the CSS thinking it will be a progressive enhancement. "It will work tomorrow".

----

I think that including features that were only documented but never shipped is still within the spirit of this rule :)

Reply:
> I've encountered this multiple times :)

Do you have examples of such cases?

---

> I think that including features that were only documented but never shipped is still within the spirit of this rule :)

Ill add it back since you feel strongly about it.